src/libgraphviz/lib.rs

```diff
@@ -795,7 +806,8 @@ r#"digraph single_edge {
     #[test]
     fn test_some_labelled() {
         let labels : Trivial = SomeNodesLabelled(vec![Some("A"), None]);
-        let result = test_input(LabelledGraph::new("test_some_labelled", labels,
+        let result = test_input(LabelledGraph::new("test_some_labelled",
+                                                   labels,
```

Comment:
Is this a rule that "either all parameters must be on their respective line, or they must share one"?